sdk/core/azure-core/test/ut/bearer_token_authentication_policy_test.cpp

```diff
@@ -171,7 +177,9 @@ TEST(BearerTokenAuthenticationPolicy, RefreshAfterExpiry)
   policies.emplace_back(
       std::make_unique<Azure::Core::Http::Policies::_internal::BearerTokenAuthenticationPolicy>(
           std::make_shared<TestTokenCredential>(accessToken),
-          Azure::Core::Credentials::TokenRequestContext{{"https://microsoft.com/.default"}}));
+          Azure::Core::Credentials::TokenRequestContext{
+              {"https://microsoft.com/.default"},
+          }));
```

Comment:
Are changes to this file only formatting?  Are we adding coverage for the new policy?